react-app/src/ExerciseSteps/ExerciseNav.js

```diff
@@ -0,0 +1,52 @@
+import { StyleSheet, View, Dimensions, Pressable } from 'react-native'
+import { useNavigation } from '@react-navigation/native'
+import React from 'react'
+import ArrowLeftIcon from '../../assets/icons/arrow-tiny-left.svg'
+import ArrowRightIcon from '../../assets/icons/arrow-tiny-right.svg'
+import TickIcon from '../../assets/icons/tick.svg'
+
+export default function ExerciseNav({ step, setStep }) {
+  const navigation = useNavigation()
+  const prevStep = () => {
+    step !== 1 && setStep(step - 1)
+  }
+  const nextStep = () => {
+    step == 8 ? navigation.navigate('Finish') : setStep(step + 1)
```

Comment:
Can we count the number of moves exists in the exercise? In this way we dont hardcode the value and when we add new moves to exercise we dont have to update here

Reply:
You are totally right!